src/components/data/DataTable.js

```diff
@@ -346,6 +372,12 @@ const DataTable = props => {
               }
             }
 
+            const filterBreadCrumb = h(div, { style: { display: 'flex', overflow: 'hidden' } }, [h(TextCell, { style: { fontWeight: 400, ...Style.noWrapEllipsis } }, [`filtered by: ${filterColTerm}`]), h(Clickable, {
+              'aria-label': 'Clear filter',
+              tooltip: 'Clear filter',
+              style: { alignSelf: 'flex-start', marginLeft: 'auto' },
```

Comment:
for the "x" icon spacing, try this:
```suggestion
              style: { alignSelf: 'flex-start', marginLeft: '0.3rem' },
```

Reply:
Aha!  That did it!  Thank you!